controllers/assignments.py

```diff
@@ -529,13 +528,13 @@ def _autograde_one_q(course_name, assignment_id, sid, qname, points, deadline=No
     question = db(db.questions.name == qname).select().first()
 
     # dispatch on grading_type; if none specified, can't autograde
-    if question.autograde == 'unittest':
+    if autograde == 'unittest':
```

Comment:
OK. But I think that's just not implemented at this time. I suggest that we make that a to-do item. (Unless you think it's actually working and this will break it.)